lib/invoice.rb

```diff
@@ -1,7 +1,8 @@
 require_relative 'date_formatter'
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.